OutSystems/outsystems-datagrid-reactive
# ROU-3735: Client action that returns the selection sum

This PR is for add new GetSelectionSum action.

### Test Steps
1. Go to API test page
2. Select multiple cells in grid
3. Click on Get Selection Sum button
4. Check if the Feedback message shows the sum of numeric cells selected.

![Animation](https://user-images.githubusercontent.com/108938618/192556077-05092123-94b8-4a33-8367-6b0789483f7d.gif)

### Checklist
